### PR TITLE
Adds ancient cryopods

### PR DESCRIPTION
**Pull request by beesil**
image

Adds ancient cryopods, a machine that will poll ghosts to become the occupant of a pod frozen in time. **Adds the pod as a large find from xenoarchaeology, a spawn in the FTL ship vault, and a semi-rare random event where it is thrown at the station.**

It has to be activated by a living mob to start recruiting. The occupant gets to pick their name, but their role & appearance is randomized. 

### Current Roles:
- cowboy
- pirate
- samurai
- prisoner
- roman
- tourist
- cosmonaut
- gangster
- pizza delivery man
- sports fan

### TO DO:
- add more roles as requested
- specific sprite while recruiting
- probably a lot of code review

:cl:
 * rscadd: Adds ancient cryopods that polls ghosts to become the occupant of a pod frozen in time. Able to be acquired through a vault, random event, and xenoarchaeology.

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
1% chance to spawn in a blue jumpsuit when

File: code/game/objects/structures/ancient_cryopod.dm [hunk 1]
```diff
@@ -0,0 +1,138 @@
+obj/machinery/cryopod
```
Comment (ghost) [inline code/game/objects/structures/ancient_cryopod.dm]: missing opening /

Comment (beesil) [reply code/game/objects/structures/ancient_cryopod.dm]: added! nice catch

**Comment by SootyBear42** [comment]
Giving people more options to keep playing after their roundstart role is dead permanently can only be good really.

Also maybe add a chance that a certain amount of Pods could be already activated at roundstart?

**Comment by SootyBear42** [comment]
I actually really want this to be added.

**Comment by SootyBear42** [comment]
Possibly add a cryogenically preserved bounty hunter

**Pull request merged**